hieradata/cluster/azar.yaml

```diff
@@ -51,14 +51,14 @@ nfs::client_mounts:
     server: "nfs1.cp.lsst.org"
     atboot: true
   /mnt/dmcs/data:
-    share: "/data/staging/dbb"
+    share: "dbb"
```

Comment:
It is due to the behavior of the the way on the nfs mount is setting up exports.  It is correctly trying to avoid overlapping bind mounts in the export path by using the lowest dir name as the share name.